easybuild/easyconfigs/s/SearchGUI/SearchGUI-3.3.3-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,35 @@
+name = 'SearchGUI'
+version = '3.3.3'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/compomics/searchgui'
+description = "SearchGUI runs proteomics identification search engines and de novo sequencing algorithms, currently X! Tandem,
+		MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA, Novor and DirecTag."
```

Comment:
@korbini You'll need to use `"""` when wrapping a string across multiple lines.

You can use `eb --check-style` to make sure your easyconfig matches the style requirements that are checked automatically.

One thing I just noticed: there's no `easyblock` specified, and there's no custom easyblock for `SearchGUI` (yet)... Was that omitted by accident maybe?

Reply:
@boegel Thank you for pointing that out.  Hope it's fine now.